### PR TITLE
fix and prettify challenge page

### PR DESCRIPTION
**Pull request by pbochis**
Added nice empty state for console
http://prntscr.com/bphliw

### CODE REVIEW

File: app/elements/coder/challenge/tasks/endpoints/ccc-io-task.html [hunk 1]
```diff
@@ -5,10 +5,10 @@
 <link rel="import" href="/elements/coder/input-view/input-view.html">
```
Comment (victorbalan) [inline app/elements/coder/challenge/tasks/endpoints/ccc-io-task.html]: remove element imports

File: app/elements/coder/code-editor/file-explorer/file-explorer-item.html [hunk 2]
```diff
@@ -32,9 +33,9 @@
 			</template>
 		</div>
 		<paper-toast id="toast"></paper-toast>
-		<paper-dialog>
-			<h1>{{dialogTitle}}</h1>
-			<input type="text" on-keyup="_handleEvent" autofocus>
+		<paper-dialog modal>
+			<h3>{{dialogTitle}}</h3>
+			<paper-input on-keyup="_handleEvent" autofocus></paper-input>
```
Comment (victorbalan) [inline app/elements/coder/code-editor/file-explorer/file-explorer-item.html]: add label

File: app/elements/company/challenge-template/challenge-template-detail.html [hunk 3]
```diff
@@ -43,15 +43,12 @@
...
-						<dd>{{duration}}</dd>
-						<dt>Description</dt>
-						<dd>{{challengeTemplate.description}}</dd>
-						<dt>Instructions</dt>
-						<dd>{{challengeTemplate.instructions}}</dd>
-					</dl>
+					<h2>{{challengeTemplate.name}}</h2>
+					<description-list>
+						<description-list-item label="Duration" value="[[duration]]"></description-list-item>
+						<description-list-item label="Description" value="[[challengeTemplate.description]]"></description-list-item>
```
Comment (victorbalan) [inline app/elements/company/challenge-template/challenge-template-detail.html]: should be markdown

File: app/elements/company/challenge-template/challenge-template-detail.html [hunk 4]
```diff
@@ -43,15 +43,12 @@
...
+						<description-list-item label="Instructions" value="[[challengeTemplate.instructions]]"></description-list-item>
```
Comment (victorbalan) [inline app/elements/company/challenge-template/challenge-template-detail.html]: should be markdown

**Comment by lorenzleutgeb** [comment]
Cool!

**Pull request merged**